client/spec/helpers/workspace.js

```diff
@@ -67,6 +68,94 @@ function Workspace() {
     };
 
     /**
+     * Show name list from right menu
+     *
+     * @param {string} name
+     */
+    this.showList = function(name) {
+        this.sideMenu.element(by.css('[title="' + name + '"]')).click();
+    };
+
+    /**
+     * Show the name highlight list
+     *
+     * @param {string} name
+     */
```

Comment:
Yes, agree. Currently the rules are document all, so already proposed a discussion about this.